src/manifest.json

```diff
@@ -30,7 +30,7 @@
     }
   },
 
-  "content_security_policy": "script-src 'self' 'sha256-MmZkN2QaIHhfRWPZ8TVRjijTn5Ci1iEabtTEWrt9CCo='; default-src 'self'; base-uri moz-extension://*;",
+  "content_security_policy": "script-src 'self' 'sha256-MmZkN2QaIHhfRWPZ8TVRjijTn5Ci1iEabtTEWrt9CCo='; default-src 'self'; base-uri moz-extension://*; object-src 'none'",
 
```

Comment:
I've noticed this has been added in m-c repository

Reply:
yeah, looks like it was added here https://bugzilla.mozilla.org/show_bug.cgi?id=1499354